applications/starlay_finance.md

```diff
@@ -0,0 +1,143 @@
+
+- Deposit: Users can deposit their preferred listed assets, receive lTokens (similar to Compound's cTokens), and earn lending fee income from the interest generated by borrowers.
+- Redeem: Users can withdraw their deposited assets and any accrued interest by redeeming their lTokens.
+- Borrow: Users can borrow assets at competitive interest rates by providing their assets as collateral, creating a seamless borrowing experience.
+- Repay: Borrowers can use this feature to repay their loans, either partially or in full, helping them manage their debt effectively.
+- Liquidate: In the event of an undercollateralized loan, users can participate in liquidations, partially repaying the loan and receiving a portion of the collateral in return.
+- Flash Loan: Developers can take advantage of flash loans, which allow them to borrow all available assets without collateral, as long as they borrow and repay within a single transaction. This feature enables innovative use cases like arbitrage, refinancing, and liquidations.
+
+
+Dashboard Overview (as the same UI as Starlay Finance)
+![starlay_dashboard](https://i.imgur.com/rXX70kY.png "UI")
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+Shumpei Koike (CEO)
+
+Seiya Chida (COO)
+
+David Lee (Lead Dev)
+
+And three other devs
+
+
+### Contact
+
+- **Contact Name:** Seiya Chida
+- **Contact Email:** seiya@starlay.finance
+- **Website:** https://starlay.finance/
+
+### Team's experience
+
+We are Starlay Finance, a team of experienced DeFi developers, designers and marketers. Starlay Finance once ranked as a huge DeFi locked with more than $500M in just 5 weeks after its launch on Astar Network. We've also developed Kagla(Stable swap) and Muuu(Yield booster) on Astar Network. 
+
+### Team Code Repos
+
+- https://github.com/starlay-finance/<Starlay Finance>
+- https://github.com/kagla-finance/<Kagla Finance>
+- https://github.com/muuu-finance/<Muuu Finance>
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/artat9
+- https://github.com/un-ast
+- https://github.com/wolfwarrier14
+- https://github.com/qudacki
+- https://github.com/chidasan
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/shumpeikoike/
+- https://www.linkedin.com/in/seiya-chida-7a9717165/
+
+## Development Status :open_book:
+
+We have completed the basic implementation and are currently adding test code and making minor modifications. As Grant progresses, we plan to request an audit firm to conduct a code audit.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 12 weeks
+- **Full-Time Equivalent (FTE):**  2FTE, 4PTE
+- **Total Costs:** 180,000 - 230,000 USD
+
+### Milestone 1 — Basic functionality
+
+- **Estimated duration:** 5 weeks
+- **FTE:**  2FTE, 4PTE
+- **Costs:** 50,000 USD
```

Comment:
So I would split this milestone into two smaller ones. I'd want to see something smaller delivered first before dolling out $50k